Orchestrator/docs/NLRModels.md

```diff
@@ -0,0 +1,29 @@
+## 									-- DRAFT --
+
+
+
+# Natural Language Representation Model
```

Comment:
I'd remove this entire section (NLR Model).

Reply:
removed - left shell to be filled.